test/parallel_api/algorithm/alg.sorting/alg.min.max/max_init_list.pass.cpp

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
```

Comment:
It looks that we have wrong file "header" here and everywhere below in new files

Reply:
Fixed.